components/Item/ItemGrid.vue

```diff
@@ -56,44 +73,560 @@ export default Vue.extend({
     loading: {
       type: Boolean,
       required: true
+    },
+    /**
+     * This is just an estimation of the *minimum* amount of cards that will be rendered: for the best experience,
+     * all the rows must contain the same amount of cards and additional cards will be created
+     * if the page is zoomed out. If any of both conditions take place, additional components to supply the carencies will be appended.
+     */
+    targetCards: {
+      type: Number,
+      required: false,
+      default: 150
```

Comment:
I tested the following values: **80, 150, 200**. 80 gives the best performance while having enough buffer space for light scrolling, 150 seems the sweetest spot for me and 200 gives near native scrolling appearance but, ofc, it's the most intensive of all the others.

Test your values and report back